MekHQ/src/mekhq/campaign/personnel/Person.java

```diff
@@ -277,6 +279,11 @@
     private int rankSystem = -1;
     private Ranks ranks;
 
+    // AwardNames
+    protected AwardsFactory awardsFactory;
+    protected ArrayList<Award> awards;
```

Comment:
Use the more generic interface type (List) as the field type rather than a specific implementation.
Is there some reason this needs to be protected rather than private?